mavros/src/plugins/sys_status.cpp

```diff
@@ -614,6 +618,18 @@ class SystemStatusPlugin : public MavRosPlugin
 		hb_diag.tick(hb.type, hb.autopilot, state_msg->mode, hb.system_status);
 	}
 
+	void handle_extended_sys_state(const mavlink_message_t *msg, uint8_t sysid, uint8_t compid) {
+		mavlink_extended_sys_state_t state;
+		mavlink_msg_extended_sys_state_decode(msg, &state);
+
+		auto state_msg = boost::make_shared<mavros_msgs::ExtendedState>();
+		state_msg->header.stamp = ros::Time::now();
```

Comment:
Ok, I was wondering because the other status messages I looked at had none ;) . So it sounds like we just leave this off in that case and deal with it properly on mavlink 2